### DISCUSSION TITLE
DNS transfer after removing Team

### DISCUSSION THREAD

**Discussion by thebergamo** [vercel/community#5659] [Account Issue] [answered]
Hi there!

After I removed my team from Vercel, I forgot to transfer my domain, so now I can transfer it to my personal account.

Could you advice what to do?

**Comment by swarnava** [comment, marked as answer]
Hi,

If you delete a team without first transferring the domain then it will remove the domain from all accounts. To re-add this, you should be able to go to your Vercel Dashboard on your Hobby account, then go to the Domains tab (https://vercel.com/dashboard/domains) and press Add. Then add the domain and it should re-add it to your new account.

**Reply by thebergamo** [reply]
Just in case if you're in the same situation, you must remove the domain from your project and add it again.

That should fix your DNS
